src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,187 @@
+
+<figure class="w-figure w-figure--center">
+  <img src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/weather_app.jpg" alt="Translucent weather card design">
+  <figcaption class="w-figcaption">
+    <a href="https://dribbble.com/shots/733714-Weather-App?list=tags&tag=android" target="_blank">Source</a>
+  </figcaption>
+</figure>
+
+Translucency, vibrancy, and blurring are effective ways of creating depth while keeping context of background content. `backdrop-filter` dynamically blends foreground and background colors based on filter functions. The result helps foreground content pop off the background content.
+
+You may recognize the effect from two popular operating systems: [Windows Vista](https://i.kinja-img.com/gawker-media/image/upload/s--9RLXARU4--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/trgz8yivyyqrpcnwscu5.png) and [iOS](https://static.businessinsider.com/image/51fd2822eab8eae16e00000b-750.jpg). Now [Chrome](https://www.chromestatus.com/feature/5679432723333120) can do it too.
+
+The following video demonstrates `backdrop-filter` and each individual filter:
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-kitchen_sink.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    <a href="https://codepen.io/robinrendle/pen/LmzLEL" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Technical Details
+{% Aside 'key-term' %}
+  backdrop - the painted content behind the element.
+{% endAside %}
+
+CSS `backdrop-filter` applies one or more filters to the backdrop of an element, leaving the contents unchanged.
+
+In order for the backdrop-filter to have any effect, some portion of the overlaying element needs to be transparent. Therefore, overlaying elements without any transparency will produce no backdrop-filter effects.
+
+<figure class="w-figure w-figure--center">
+  <img src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop-filter.png" alt="2 comparisons where one is opaque and the other has transparency and therefore blurred translucency">
+</figure>
+
+Backdrop-filters are like CSS filters. When the property is not 'none', a new [stacking context](https://www.w3.org/TR/CSS21/zindex.html) and/or [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) are created. Note, only elements with absolute and fixed position descendants will get a containing block.
```

Comment:
These are pretty much the only technical details in the whole article, and it's 1 paragraph of niche complexity about edge cases and footguns. I feel they need to be there so it's not just a pretty pictures article because it offers a tiny glimpse into the advanced knowledge of the property. thoughts? i'm semantically ok with putting the comments into an aside, but visually and academically i dont think it's a good idea. 🤷‍♂️ 

when reading about a new css property, wouldnt you want to know the details about common foot guns and quick tidbits that'll save you hours later?

Reply:
Right, I didn't mean to suggest this be _omitted_; I agree that it's important for readers to understand common pitfalls. My worry is that the post doesn't yet make explicit what those pitfalls _are_. For example, why is important to know that this property creates a new stacking context and containing block? What problems might that cause for you if you weren't expecting it? I think that'll make this info more actionable for newer devs.